components/collector/src/source_collectors/harbor_json/security_warnings.py

```diff
@@ -0,0 +1,63 @@
+"""Harbor JSON security warnings collector."""
+
+from typing import Final, TypedDict, cast
+
+from base_collectors import JSONFileSourceCollector
+from collector_utilities.type import JSON
+from model import Entities, Entity
+
+REPORT_MIME_TYPE: Final = "application/vnd.security.vulnerability.report; version=1.1"
```

Comment:
Hmm, interesting idea. Let's hope we won't need to support multiple versions, however :-)